### ISSUE TITLE
PI: A parameter can be added multiple times if linked with other parameters.

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#295]
One can add a parameter for the identification multiple times if it is linked to other parameters. E.g., parameter "A" can be added to the list of identification parameters as a distinct parameter, and, additionally, it can be linked to parameters "B", "C", etc.

I think a check should be performed that would block adding a parameter more than once, as it is absolutely not clear, how  the parameter will be identified.

**Comment by UlrichSi** [comment]
I guess the current philosophy sets flexibility above having everything fool-proof. While it certainly may not hurt to do such automatic check, there are just a lot of things in MoBi where a user can screw things up - so what you found may just be one more of these situations.

**Comment by msevestre** [comment]
Yes this is something that we could easily prevent from happening

**Comment by PavelBal** [comment]
>Yes this is something that we could easily prevent from happening

For now - can you tell, which value is actually used? :D

**Comment by msevestre** [comment]
not without digging in the code and trying it out. I would assume that the LAST one in the list is the one used but who knows :) 

This could easily be checked by creating a PI with absurd values for the first entry and see if the results are changing
